build.sbt

```diff
@@ -5,14 +5,14 @@ name := "akka-testkit-specs2"
 organization := "net.ruippeixotog"
 
 scalaVersion := "2.13.0"
```

Comment:
Can you update the core Scala version to be in 2.13.1, too?

Reply:
Oups, fixed!